Service/Request/RequestFactory.php

```diff
@@ -210,4 +211,17 @@ private function trimCurlyBrackets($str)
     {
         return trim($str, '{}');
     }
+
+    /**
+     * @param array $queryParamsString
+     *
+     * @return array
+     */
+    private function normalizeQueryParams($queryParamsArray)
```

Comment:
PHP7 already has type hinting for arguments and return values

Reply:
Added